foods.html

```diff
@@ -59,6 +60,49 @@
           </tbody>
         </table>
       </div>
+
+      <div class = "meal-dinner">
+        <h2>Dinner</h2>
+        <table id="dinner">
+          <thead>
+            <tr><th>Name</th><th>Calories</th></tr>
+          </thead>
+          <tbody>
+          </tbody>
+        </table>
+      </div>
+      
+      <div class = "meal-snack">
+        <h2>Snack</h2>
+        <table id="snack">
+          <thead>
+            <tr><th>Name</th><th>Calories</th></tr>
+          </thead>
+          <tbody>
+          </tbody>
+        </table>
```

Comment:
You might consider adding a `<tfoot>` as well. That way, when you're manipulating the rows, you don't include the totals.